libs/unity/library/Runtime/Scripts/NativeRender/NativeRenderer.cs

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+
+namespace Microsoft.MixedReality.WebRTC.UnityPlugin
+{
+    public delegate void LogCallback(string str);
+
+    public enum VideoKind : int
+    {
+        None = 0,
+        I420 = 1,
+        ARGB = 2
+    }
+
+    public class TextureDesc
+    {
+        public IntPtr texture;
+        public int width;
+        public int height;
+    }
+
+    /// <summary>
+    /// High-performance, low-level, native video rendering for Microsoft.MixedReality.WebRTC.PeerConnection objects.
+    /// </summary>
+    public class NativeRenderer : IDisposable
```

Comment:
Same as `NativeRendererPluginUpdate`, this should be an internal class or nested in `NativeVideoRenderer` as it only needs to be visible there. Also, the name is a bit confusing since it's not clear what the difference is between this and `NativeVideoRenderer`. It should probably be `NativeRendererProxy` or something like that.

Reply:
Renamed NativeRender to NativeVideo and NativeRendererInterop to NativeVideoInterop. I think this naming will work better. You create a NativeVideo object, and the NativeVideoRenderer renders the NativeVideo.